openpathsampling/sample.py

```diff
@@ -756,17 +760,24 @@ class Sample(StorableObject):
         The Ensemble this sample is drawn from
     """
 
+    parent = DelayedLoader()
+    # details = DelayedLoader()
+    mover = DelayedLoader()
+
     def __init__(self,
                  replica=None,
                  trajectory=None,
                  ensemble=None,
                  bias=1.0,
-                 details=None,
                  parent=None,
-                 mover=None
+                 mover=None,
+                 details=None
```

Comment:
What is the current status on `Sample.mover` and `Sample.details`? Are these both deprecated? I think they were only used in the path tree visualization, and if that has switched to using the `MoveChange` instead, then they can be deprecated (can't remove yet because of backward compatibility).